.dockerignore

```diff
@@ -1,9 +1,2 @@
 build
 test
-.git
-girder
-girder_annotation
-large_image
-sources
-utilities
-examples
```

Comment:
This might be causing problems for CircleCI

Reply:
The existing dockerfile is used for one of the test environments and shouldn't care.  It probably should be renamed (to `test.Dockerfile`?) to disambiguate the purposes for the new Dockerfile.  You should only need to change the build line in .circleci/config.yml to handle such a name change.

Reply:
And, if you think is semantically clearer, feel free to move the existing Dockerfile to some subdirectory within the project.

Reply:
I'll rename that to `test.Dockerfile` to be more clear.